.github/workflows/db_import.yml

```diff
@@ -0,0 +1,65 @@
+name: DB Import
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - devtools/db_import/**
+  # This also needs to run on push to fill the GitHub Actions Cache with the current Docker image,
+  # so that subsequent PRs can inherit this cache and leverage the prebuilt Docker image.
+  push:
+    branches:
+      - main
```

Comment:
Maybe also add `workflow_dispatch` for manual trigger?

Reply:
Done.